exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,7 @@
 function halve(number) {
-  // complete the function here
+  return number/2;
```

Comment:
It would be better if you added some spaces between the operator here, it makes it easier to read.
```
return number / 2
```